### DISCUSSION TITLE
Questions about h3_get_pentagons

### DISCUSSION THREAD

**Discussion by julsbreakdown** [bytesandbrains/h3-pg#148] [Announcements]
Hello
Using `select * from h3_get_pentagons(0) set_` gives me only 12 rows 
Whereas `select * from h3_get_res_0_cells() st_` gives all 122

The documentation states 
```
h3_get_pentagons(resolution integer) ⇒ SETOF h3index
Since v4.0.0
```

=> All the pentagon H3 indexes at the specified resolution.
So why am i getting 12 values, am I missing something?
thanks

**Comment by zachasme** [comment]
Hi @julsbreakdown, I'm assuming you found an answer on your own, but I'll convert this into a discussion for future reference, in case anyone else should have the same question.

There are 12 pentagons at every resolution, but a increasing number of cells/hexagons on each resolution. More information in the [H3 core documentation](https://h3geo.org/docs/core-library/overview/).

Related: https://github.com/zachasme/h3-pg/discussions/146
